command/agent.go

```diff
@@ -15,6 +15,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/hashicorp/consul/agent/systemd"
```

Comment:
I don't think we should be depending on an internal Consul package.  Can it be factored out somewhere instead, or if it's very small maybe we can duplicate it?

Reply:
It's rather small & I'd agree to making a copy local to this repo retaining some header of where its taken from :-) - let me know what to do.

Reply:
@ncabatoff should I localise the `systemd` dependency? - kindly suggest how to proceed with this PR.

Reply:
Yes please